src/map_server/include/map_server/base_map_loader.h

```diff
@@ -0,0 +1,37 @@
+#ifndef MAP_SERVER_BASE_MAP_LOADER_H
+#define MAP_SERVER_BASE_MAP_LOADER_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <libgen.h>
+#include <fstream>
+#include "rclcpp/rclcpp.hpp"
+
+
```

Comment:
Please remove extra whitespace. See other source files for example source formatting.